wire-ios/Wire-iOS Tests/ConversationMessageCell/ConversationTextMessageTests.swift

```diff
@@ -205,7 +205,7 @@ final class ConversationTextMessageTests: ConversationMessageSnapshotTestCase {
 
     // MARK: - Helper Methods
 
-    func createMessage(withText: String = "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem.",
```

Comment:
?

Reply:
commit https://github.com/wireapp/wire-ios/pull/789/commits/699500bfd4bc25794faedfc657db91b3e3fb964a I'll revert 
it's needed to compile on Xcode 15